ts/screens/preferences/OldServicesHomeScreen.tsx

```diff
@@ -155,7 +155,9 @@ export const getAllSections = createSelector(
       })
       .filter(_ => _.data.length > 0)
       .sort((a, b) =>
-        (a.title || "").localeCompare(b.title, "it", { caseFirst: "lower" })
+        (a.title.toLocaleLowerCase() || "").localeCompare(
```

Comment:
```suggestion
        (a.title.toLocaleLowerCase()).localeCompare(
```
you could remove double pipe because **title** is a string (not string | undefined)
how title is built:
`const title = organizations[fiscalCode] || fiscalCode;`